news/2019-07-07-beatmap-spotlights-spring-2019.md

```diff
@@ -0,0 +1,306 @@
+
+_written by [Toy](https://osu.ppy.sh/users/2757689)
+
+## <a name="taiko" id="taiko"></a>osu!taiko
+
+[![](/wiki/shared/news/2019-07-07-beatmap-spotlights-spring-2019/taiko/ylil.jpg)](https://osu.ppy.sh/beatmapsets/897948)
+
+Starting off the list of spotlighted maps for this season, we have a mapset extraordinaire coming from [SKSalt](https://osu.ppy.sh/users/3280542); a relatively old mapper that some considered to be ahead of their time back in the day. Even today this holds true, as this rendition of "Ylil's overed landscape" by xyst masterfully exemplifies!
+
+There are so many things to say about this map, such as ranging from a load of seven refined difficulties to make sure players of all skill levels are in for a treat. Amazing spread management which helps less-experienced players advance to the next stage, and last, but most certainly not least, the sheer creativity and execution you can see in the top difficulty.
+
+Speaking of which, boy, ohh boy is it hard. Insanely hard. You might expect "pp map" when you look at it's BPM and Star Rating, but let me tell you: this is anything but ordinary. The way 1/6 patterns are arranged will keep your fingers dancing constantly during the intense parts, as do the streams, which also feature a mix of longer 1/6 bursts; up to 13 notes in a row, which is not easy to keep up with at 200 BPM, especially with their rather unusual colouring! Also, the Slider Velocity manipulation will demand your eyes to adjust quickly as it gets increasingly faster along with the song's intensity, befitting it perfectly!
+
+If you're not ready for the challenge this difficulty provides, you should give [komasy](https://osu.ppy.sh/users/1980256)'s Inner Oni a shot! It's still quite challenging and an absolute joy to play through, just like the other difficulties are.
+
+
+A blooming start to this season's spotlights, and an absolutely deserved winner to it!
+
+_written by [Greenshell](https://osu.ppy.sh/users/8693851)
+
+[![](/wiki/shared/news/2019-07-07-beatmap-spotlights-spring-2019/taiko/goodworld.jpg)](https://osu.ppy.sh/beatmapsets/938075)
+
+As one of the most liked mapsets for the season, "GOODWORLD" by [komasy](https://osu.ppy.sh/users/1980256) is a perfect example of a carefully crafted mapset from the beginning and establishing a consistent style across it.
+
+While one could easily just praise komasy's good taste when it comes to patterns and the way they are emphasized by the Slider Velocity Changes he uses; it would fall short if we did not also talk about how the entire mapset consists of difficulties that are crazy good as standalone diffs. This is proof of how komasy doesn't even need an absurd level of notes to express himself.
```

Comment:
```suggestion
While one could easily just praise komasy's good taste when it comes to patterns and the way they are emphasized by the slider velocity changes he uses, it would fall short if we did not also talk about how the entire mapset consists of difficulties that are crazy good as standalone diffs. This is proof of how powerful his expression is, if nothing else.
```